journalbeat/docs/getting-started.asciidoc

```diff
@@ -211,51 +162,25 @@ start {beatname_uc} in the foreground.
 sudo service {beatname_lc} start
 ----------------------------------------------------------------------
 
-*mac:*
+*linux:*
 
 ["source","sh",subs="attributes"]
 ----------------------------------------------------------------------
 sudo chown root {beatname_lc}.yml <1>
-sudo ./{beatname_lc} -e -c {beatname_lc}.yml -d "publish"
+sudo ./{beatname_lc} -e
 ----------------------------------------------------------------------
 <1> You'll be running {beatname_uc} as root, so you need to change ownership
 of the configuration file, or run {beatname_uc} with `--strict.perms=false`
 specified. See
 {libbeat}/config-file-permissions.html[Config File Ownership and Permissions]
 in the _Beats Platform Reference_.
 
-*win:*
-
-["source","sh",subs="attributes"]
-----------------------------------------------------------------------
-PS C:{backslash}Program Files{backslash}{beatname_uc}> Start-Service {beatname_lc}
-----------------------------------------------------------------------
-
-
-By default, Windows log files are stored in +C:\ProgramData\{beatname_lc}\Logs+.
-
-{beatname_uc} is now ready to send log files to your defined output.
+{beatname_uc} is now ready to send journal events to the defined output.
 
 [[view-kibana-dashboards]]
-=== Step 6: View the sample Kibana dashboards
-
-IMPORTANT: This documentation is placeholder content. It has not yet been reviewed.
-
-To make it easier for you to explore {beatname_uc} data in Kibana, we've created
-example {beatname_uc} dashboards. You loaded the dashboards earlier when you
-ran the `setup` command.
-
-include::../../libbeat/docs/opendashboards.asciidoc[]
-
-You can use these dashboards as examples and
-{kibana-ref}/dashboard.html[customize] them to meet your needs.
-
-To populate the example dashboards with data, you need to either
-<<configuring-ingest-node,define ingest node pipelines>> or use Logstash to
-parse the data into the fields expected by the dashboards. 
+=== Step 5: View your data in Kibana
```

Comment:
I don't think we should document Kibana settings here (they'll get overlooked when the settings change in Kibana). I can do this for the initial release, though, since the docs aren't GA yet.

Reply:
You are right with the point of having the risk of overlooking settings changes on Kibana.

I think that ideally log viewer should be configured by default with the default indexes of all beats that generate logs, but we talked about that and we decided that we are not doing it for non-GA beats. So then It looked like a good idea to add a note on Beats setup doc about this Kibana configuration, but you are right, this can be too coupled to Kibana changes.

I am worried users go to try 6.5, they see that there is a new log viewer and a new journalbeat, but they don't see them working together.

What about adding a note with just a reference to the log viewer documentation?